docs/guides/text-tracks.md

```diff
@@ -1,184 +1,201 @@
+This class can be used to create new text track objects. Each property below is available as an option to the `TextTrack` constructor, as a property of the created `TextTrack` object, or as a DOM attribute.
 
-### `player.textTracks() -> TextTrackList`
-This is the main interface into the text tracks of the player.
-It return a TextTrackList which lists all the tracks on the player.
+#### `kind`
+One of the track types supported by Video.js:
 
-### `player.remoteTextTracks() -> TextTrackList`
-This is a helper method to get a list of all the tracks that were created from `track` elements or that were added to the player by the `addRemoteTextTrack` method. All these tracks are removeable from the player, where-as not all tracks from `player.textTracks()` are necessarily removeable.
+- `"subtitles"`: Translations of the dialogue in the video for when audio is available but not understood. Subtitles are shown over the video.
+- `"captions"`: Transcription of the dialogue, sound effects, musical cues, and other audio information for viewer who are deaf/hard of hearing, or the video is muted. Captions are also shown over the video.
+- `"chapters"`: Chapter titles that are used to create navigation within the video. Typically, these are in the form of a list of chapters that the viewer can use to navigate the video.
+- `"descriptions"`: Text descriptions of the action in the content for when the video portion isn't available or because the viewer is blind or not using a screen. Descriptions are read by a screen reader or turned into a separate audio track.
+- `"metadata"`: Tracks that have data meant for JavaScript to parse and do something with. These aren't shown to the user.
 
-### `player.remoteTextTrackEls() -> HTMLTrackElementList`
-Another helper method, this is a list of all the `track` elements associated with the player. Both emulated or otherwise.
+Defaults to `"subtitles"`.
 
-### `player.addTextTrack(String kind, [String label [, String language]]) -> TextTrack`
-This is based on the [w3c spec API](http://www.w3.org/html/wg/drafts/html/master/embedded-content-0.html#dom-media-addtexttrack) and when given a kind and an optional label and language, will create a new text track for you to use.
-This method is intended for purely programmatic usage of tracks and has one important limitation:
-tracks created using this method *cannot* be removed. The native `addTextTrack` does not have a corresponding `removeTextTrack`, so, we actually discourage the usage of this method.
+#### `label`
+Short descriptive text for the track that will used in the user interface. For example, in a menu for selecting a captions language.
+
+#### `default`
+The boolean `default` attribute can be used to indicate that a track's mode should start as `"showing"`. Otherwise, the viewer would need to select their language from a captions or subtitles menu.
+
+> **Note:** For chapters, `default` is required if you want the chapters menu to show.
+
+#### `srclang`
```

Comment:
The actual property is called `language`, but it can be filled by `srclang` or `language` options as options